stories/clickAnalytics.stories.js

```diff
@@ -0,0 +1,143 @@
+import { storiesOf } from '@storybook/html';
+import { withHits } from '../.storybook/decorators';
+import * as analyticsHelpers from '../src/helpers/analytics';
+
+const insightsClientMock = (eventName, payload) => {
```

Comment:
You can use the `action` from Storybook ([see Analytics story](https://github.com/algolia/instantsearch.js/blob/develop/stories/analytics.stories.js)).